### ISSUE TITLE
Compilation fails for ARM due to PTRACE_SETVFPREGS undefined

### ISSUE THREAD

**Issue by szymon-mulczynski-red** [checkpoint-restore/criu#1629]
**Description**
 compel/arch/arm/src/lib/infect.c compilation fails due to compel_set_task_ext_regs() using undefined PTRACE_SETVFPREGS.

**Steps to reproduce the issue:**
1. checkout latest CRIU on master branch
2. compile for ARM

**Describe the results you received:**
| compel/arch/arm/src/lib/infect.c: In function 'compel_set_task_ext_regs':
| compel/arch/arm/src/lib/infect.c:107:13: error: 'PTRACE_SETVFPREGS' undeclared (first use in this function)
|   if (ptrace(PTRACE_SETVFPREGS, pid, NULL, ext_regs)) {
|              ^~~~~~~~~~~~~~~~~
| compel/arch/arm/src/lib/infect.c:107:13: note: each undeclared identifier is reported only once for each function it appears in
| make[1]: *** [compel/arch/arm/src/lib/infect.o] Error 1
| make[1]: *** Waiting for unfinished jobs....

**Additional information you deem important (e.g. issue happens only occasionally):**
Equivalent value for get: PTRACE_GETVFPREGS is defined in the file right above the usage.

**Comment by szymonMulczynskiSky** [comment]
Hi,

I thought this error is rather related with the commit introducing the PTRACE_SETVFPREGS usage in this file with no direct include:

commit 21e3c530731f0f77119958a7bd26b7a954cf67d3
Author: Dmitry Safonov <dima@arista.com>
Date:   Fri Feb 26 02:25:57 2021 +0000

For some reason GET* is redefined here, while SET* was not. This looks to me like missing include or missing redefinition (to keep things consistent), but I don't know the rationale. Do you agree something should be improved here?

Thank you for the patch -> this works of course.

Now my compilation fails on strange error (Error: invalid constant (1b3) after fixup). I think it's worth another issue if I don't find the solution.
